crates/locutus-node/src/operations/subscribe.rs

```diff
@@ -55,27 +55,9 @@ impl StateMachineImpl for SubscribeOpSm {
                     retries: 0,
                 })
             }
-            (SubscribeState::ReceivedRequest, SubscribeMsg::SeekNode { found: true, .. }) => {
+            (SubscribeState::ReceivedRequest, SubscribeMsg::SeekNode { .. }) => {
```

Comment:
SeekNode { found: false } cannot happen in the original requester right?

Reply:
this should not happen, the original requester would get a `ReturnSub`